channeldb/channel.go

```diff
@@ -2568,6 +2574,11 @@ func (h *HTLC) serializeExtraData() error {
 		records = append(records, &b)
 	})
 
+	records, err := h.CustomRecords.ExtendRecordProducers(records)
```

Comment:
Add test coverage for serialization change?

Reply:
Added that to the [tracking issue](https://github.com/lightningnetwork/lnd/issues/8769).